gui/src/controls/multiLine.ts

```diff
@@ -0,0 +1,215 @@
+
+        private _lineWidth: number = 1;
+        private _dash: number[];
+        private _points: Nullable<MultiLinePoint>[];
+
+        private _minX: Nullable<number>;
+        private _minY: Nullable<number>;
+        private _maxX: Nullable<number>;
+        private _maxY: Nullable<number>;
+
+        constructor(public name?: string) {
+            super(name);
+
+            this.isHitTestVisible = false;
+            this._horizontalAlignment = Control.HORIZONTAL_ALIGNMENT_LEFT;
+            this._verticalAlignment = Control.VERTICAL_ALIGNMENT_TOP;
+
+            this._dash = [];
+            this._points = [];
+        }
+
+        public get dash(): Array<number> {
+            return this._dash;
+        }
+
+        public set dash(value: Array<number>) {
+            if (this._dash === value) {
+                return;
+            }
+
+            this._dash = value;
+            this._markAsDirty();
+        }
+
+        getAt(index: number): MultiLinePoint {
+            if (!this._points[index]) {
+                this._points[index] = new MultiLinePoint(this);
+            }
+
+            return this._points[index] as MultiLinePoint;
+        }
+
+        onPointUpdate = (): void => {
+            this._markAsDirty();
+        }
+
+        add(...items: (AbstractMesh | Control | IValueAndUnitVector2)[]): void {
+            items.forEach(item => {
+                var point: MultiLinePoint = this.push();
+
+                if (item instanceof AbstractMesh) {
+                    point.mesh = item;
+                }
+                else if (item instanceof Control) {
+                    point.control = item;
+                }
+                else if (item.x != null && item.y != null) {
+                    point.x = item.x;
+                    point.y = item.y;
+                }
+            });
+        }
+
+        push(): MultiLinePoint {
+            return this.getAt(this._points.length);
+        }
+
+        remove(value: number | MultiLinePoint): void {
+            var index: number;
+            
+            if (value instanceof MultiLinePoint) {
+                index = this._points.indexOf(value);
+
+                if (index === -1) {
+                    return;
+                }
+            }
+            else {
+                index = value;
+            }
+            
+            var point: Nullable<MultiLinePoint> = this._points[index];
+
+            if (!point) {
+                return;
+            }
+
+            point.dispose();
+
+            this._points.splice(index, 1);
+        }
+        
+        public get lineWidth(): number {
+            return this._lineWidth;
+        }
+
+        public set lineWidth(value: number) {
+            if (this._lineWidth === value) {
+                return;
+            }
+
+            this._lineWidth = value;
+            this._markAsDirty();
+        }   
+
+        public set horizontalAlignment(value: number) {
+            return;

```

Comment:
?

Reply:
I started this class from a copy of Line. I'm not sure what's the reason behind it:
https://github.com/BabylonJS/Babylon.js/blob/master/gui/src/controls/line.ts#L118

Reply:
Oh yeah true :)